pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -906,7 +906,7 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		if network.Pod != nil {
 			numberOfSources++
 		}
-		if network.Multus != nil {
+		if network.Npwg != nil || network.Multus != nil {
```

Comment:
(No action required) looks like we are doing this type of check in multiple places. This could be a helper function like ```isMultusNetwork``` that would do that.

Reply:
Done

Reply:
it looks like this is the same as before still?! maybe I'm missing something?